File: examples/pascal/constructs/string_mixed.pas
// Ported from examples/lumen/constructs/string_mixed.lm by scripts/port_examples.py; edit the Lumen original, not this file.
var message: string;
var n: integer;
begin
    message := 'value: ';
    n := 42;
    writeln(message);
    writeln(n);
    if n = 42 then begin
        writeln('correct');
    end;
end.

File: examples/rust/constructs/string_mixed.rs
// Ported from examples/lumen/constructs/string_mixed.lm by scripts/port_examples.py; edit the Lumen original, not this file.
fn main() {
    let message = "value: ";
    let n = 42;
    println!("{}", message);
    println!("{}", n);
    if n == 42 {
        println!("correct");
    }
}
